System/Metrics.hs

```diff
@@ -445,6 +445,7 @@ getGcStats = do
 
 -- | Empty GC statistics, as if the application hasn't started yet.
 emptyGCStats :: Stats.GCStats
+#if MIN_VERSION_base(4,10,0)
```

Comment:
When I nest ifdefs I usually indent the nested one one space for readability. Could you change this line to

```
# if MIN_VERSION_base(4,10,0)
```

and similarly for any `else` and `endif`.